plaso/parsers/logd.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Parser for the logd.0.log file obtained from iOS Sysdiagnose file dumps."""
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class LogdEventData(events.EventData):
+  """logd event data
+
+  Attributes:
+    logger (str): process that submits the log event
+    body (str): body of the event line
+  """
+
+  DATA_TYPE = "logd:line"
+
+  def __init__(self):
+    """Initializes event data."""
+    super(LogdEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.logger = None
+    self.body = None
+
+
+class LogdParser(text_parser.PyparsingSingleLineTextParser):
+  """Parser for the logd.0.log file obtained from iOS Sysdiagnose file dumps."""
+
+  NAME = 'logd.0.log'
+  DATA_FORMAT = 'logd log'
```

Comment:
Minor comment but I'd suggest to mention this is for iOS sysdiagnose logs as part of the description. 
```
DATA_FORMAT = 'iOS sysdiagnose log file'
```